Formula/diffuse.rb

```diff
@@ -23,7 +23,14 @@ def install
   end
 
   test do
+    require "pty"
+
     system "#{bin}/diffuse", "--help"
     assert_match version.to_s, shell_output("#{bin}/diffuse --version")
+
+    _, _, pid = PTY.spawn "#{bin}/diffuse"
```

Comment:
We should get this to dump a log and then check that for errors. Otherwise I think `PTY.spawn` will ignore any errors that occur.